src/main/java/com/example/eatmate/app/domain/report/dto/ReportRequestDto.java

```diff
@@ -0,0 +1,19 @@
+package com.example.eatmate.app.domain.report.dto;
+
+import java.util.List;
+
+import com.example.eatmate.app.domain.report.domain.ReportType;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.Getter;
+
+@Getter
+public class ReportRequestDto {
+	@NotEmpty
+	private List<ReportType> reportTypes;
+	@NotBlank
+	private String reportingReasonDescription;    // 구체적인 신고 사유
+	@NotBlank
+	private String reportedMemberEmail;
```

Comment:
```suggestion
	@NotEmpty
	private List<ReportType> reportTypes;
	
	@NotBlank
	private String reportingReasonDescription;    // 구체적인 신고 사유
	
	@NotBlank
	private String reportedMemberEmail;
```

한줄씩 띄워주시면 좋을 것 같습니다!

Reply:
수정했습니다!